drivers/sensor/lsm6dsl/Kconfig

```diff
@@ -39,6 +39,24 @@ endchoice
 config LSM6DSL_TRIGGER
 	bool
 
+config LSM6DSL_TRIGGER_DEFAULT
+	int "Default trigger"
+	depends on LSM6DSL_TRIGGER
+	range 0 1
+	default 0
+	help
+	  Specify the default trigger at startup.
+	  0: Data ready
+	  1: Significant Motion
+
+config LSM6DSL_TRIGGER_SIGN_MOTION_THRESH
+	int "Threshold for significant motion detection"
+	depends on LSM6DSL_TRIGGER
+	range 0 255
+	default 6
+	help
+	  Specify the threshold for significant motion detection
```

Comment:
These should be devicetree properties

Reply:
Thanks, I'll do that when I have some time. For my information, are there guidelines as to what should be devicetree properties and what should be KConfig options ?

Reply:
I think that would be nice to change properties in ```st,lsm6dsl-common.yaml``` in order to have two interrupts gpios, i.e. int1-gpios and int2-gpios, so that we can have drdy on one and interrupts on the second.